examples/counter-next/shared/uniffi.toml

```diff
@@ -0,0 +1,7 @@
+[bindings.kotlin]
+package_name = "com.crux.example.counter.shared"
+cdylib_name = "shared"
+
+[bindings.swift]
+cdylib_name = "shared_ffi"
+omit_argument_labels = true
```

Comment:
More out of curiosity - does this still get read by the uniffi implementation which the crux_cli is wrapping?

Reply:
I think so. But there are some helpful comments above (by @srfsh) on how we can avoid this. I'm going to visit this configuration and how we best provide it in a new PR